sycl/source/detail/graph_impl.hpp

```diff
@@ -869,6 +870,20 @@ class exec_graph_impl {
     return false;
   }
 
+  /// Checks if the previous submissions of this graph have been completed
+  /// This function checks the status of events associated to the previous graph
+  /// submissions.
+  /// @return true if all previous submissions have been completed, false
+  /// otherwise.
+  bool arePreviousSubmissionsCompleted() const {
```

Comment:
How about renaming into `allPreviousSubmissionsCompleted` or just `previousSubmissionCompleted` ?

Reply:
I changed the name to previousSubmissionCompleted